website/src/pages/customers.tsx

```diff
@@ -241,7 +241,7 @@ export default ((props: any) => (
                         <p>Watch how Sid Sijbrandij (GitLab CEO) and Quinn Slack (Sourcegraph CEO) do code reviews.</p>
                     </div>
                     <div className="row justify-content-center pt-md-4">
-                        <div className="col-lg-10 text-center">
+                        <div className="col-lg-10">
```

Comment:
After this change, the logo for Workiva is left aligned, whereas all the other logos are still centered. I think there's a fix needed where the `img` tag needs to be [wrapped in a div, like the other logos](https://github.com/sourcegraph/about/pull/2872/files#diff-ad3c9221af8145935b3f59e56855ef1ea64b6af665c8552afb63d8c42acf8669R314)

Reply:
good catch. thank you. updated it.

Reply:
Thanks guys. This request was from me, we started an Asana board to track our requests to Mate for the marketing site.